src/main/java/frc/robot/motorcontrol/HallEffectMagnet.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.motorcontrol;
```

Comment:
These guys are still in `motorcontrol` when the folder was renamed to `util` a while back. Maybe move it there?

Reply:
idt it makes sense for magnets to be in a motor controller folder